packages/datepicker/src/react/__stories__/index.story.tsx

```diff
@@ -141,7 +142,40 @@ storiesOf('SingleDate', module)
       </div>
     )
   })
-  .add('DatePicker', () => <DatePicker />)
+  .add('DatePicker', () => (
```

Comment:
nitpick - this should be it's own top level story category

![Screen Shot 2021-03-04 at 2 42 31 PM](https://user-images.githubusercontent.com/226356/110034631-ee32b900-7cf7-11eb-9bbe-8cd644dfe7a8.png)